rustfmt.toml

```
@@ -0,0 +1,2 @@
+edition = "2021"
```

`cargo fmt` doesn't need it, because it gets the edition from `Cargo.toml`. I'm assuming calling `rustfmt` directly doesn't do that?

Lets just leave a comment to that effect here:
```suggestion
// Required to make direct calls to `rustfmt` work for our WebIDL generation.
edition = "2021"
```

---

> I'm assuming calling rustfmt directly doesn't do that?

I have no idea. I just wanted to get rid of the `.arg("edition").arg("2021")`